### PR TITLE
updated html and css files with code

### PR DESCRIPTION
**Pull request by jairatagle**
## Description
This PR creates HTML structure and updated with comments. Starts CSS file with starter code.

## Acceptance Criteria
- HTML skeleton created
- CSS file created
- Added styling for root, *, and nav

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+h1 {
+    font-size: 48px;
+}
+
+/* ============================================== */
+
+/* Search box styling */
+
+#search-window {
+    background-color: yellow;
```
Comment (ANAZ0004) [inline assets/css/style.css]: Can you change the background color to --white-bg-color?
```suggestion
    background-color: var(--white-bg-color);
```

File: index.html [hunk 2]
```diff
@@ -6,6 +6,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Weather Dashboard Project</title>
```
Comment (ANAZ0004) [inline index.html:7]: @dosanjhkiran Can you think of a new title?

**Review by ANAZ0004 — CHANGES_REQUESTED** [review]

File: assets/css/style.css (hunk 1 above)
Comment (jairatagle) [reply assets/css/style.css]: updated the bg color!

File: index.html (hunk 2 above)
Comment (jairatagle) [reply index.html:7]: We can add the title later when it's finalised. :)

**Review by ANAZ0004 — APPROVED** [review]
LGTM!

**Pull request merged**